llvm/lib/Target/RISCV/RISCVCodeGenPrepare.cpp

```diff
@@ -155,6 +163,47 @@ bool RISCVCodeGenPrepare::visitIntrinsicInst(IntrinsicInst &I) {
   return true;
 }
 
+bool RISCVCodeGenPrepare::expandVPStrideLoad(IntrinsicInst &II) {
+  if (ST->hasOptimizedZeroStrideLoad())
```

Comment:
We don't have test coverage for this?

Reply:
I think we need to negate this condition now

Reply:
Oops ignore my previous comment, I got confused